src/main/java/it/gov/pagopa/reward/notification/dto/controller/detail/RefundDetailDTO.java

```diff
@@ -0,0 +1,32 @@
+package it.gov.pagopa.reward.notification.dto.controller.detail;
+
+import it.gov.pagopa.reward.notification.enums.RewardNotificationStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class RefundDetailDTO {
```

Comment:
This entity ha RewardNotification name inside this application
```suggestion
public class RewardNotificationDetailDTO {
```